apps/drive-app/src/pages/PhotoCapturePage/PhotoCapturePage.tsx

```diff
@@ -51,6 +58,7 @@ export function PhotoCapturePage() {
         useLiveCompliance={config.useLiveCompliance}
         customComplianceThresholds={config.customComplianceThresholds}
         customComplianceThresholdsPerSight={config.customComplianceThresholdsPerSight}
+        validateButtonLabel={validateButtonLabel}
```

Comment:
So I think the translation label should just be a string, and we add the translation in the application JSON files, and we pass the prop like this :

```tsx
export function PhotoCapturePage() {
  const { t } = useTranslation();

  return (
    <PhotoCapture
      validateButtonLabel={t('photo-capture.validate-label')}
    />
  );
}
```